src/olympia/abuse/models.py

```diff
@@ -210,8 +210,12 @@ def target(self):
     @property
     def type(self):
         with translation.override(settings.LANGUAGE_CODE):
-            type_ = (translation.ugettext(amo.ADDON_TYPE[self.addon.type])
-                     if self.addon else 'User' if self.user else 'Addon')
+            if self.addon and self.addon.type in amo.ADDON_TYPE:
+                type_ = (translation.ugettext(amo.ADDON_TYPE[self.addon.type]))
```

Comment:
🤔 why _do_ we have this literal `translation.ugettext(amo.ADDON_TYPE[self.addon.type])` rather than `amo.ADDON_TYPE[self.addon.type]`?

Reply:
To ensure it's... *not* translated in whatever the current language is in the current thread but instead rendered in english (it's inside a `translation.activate()`, the tests cover that)

Reply:
ah, missed that.